### ISSUE TITLE
Intermittent test failures for "ag-solo on xs / build"

### ISSUE THREAD

**Issue by katelynsills** [Agoric/agoric-sdk#442]
I've been getting intermittent test failures for "ag-solo on xs / build", even on PRs that shouldn't cause any difference, such as https://github.com/Agoric/agoric-sdk/pull/441/checks?check_run_id=403791125

**Comment by dckc** [comment]
Yes; these symptoms look familiar. My (unconfirmed) diagnosis is: gcc occasionally blows chunks. Running it again works pretty well as a treatment, in my experience.

```
/home/runner/moddable/build/tmp/lin/release/marshal/mc.xs.c:1172:2: error: expected expression at end of input
  0x3d, 0x4f, 0xff, 0xff, 0x2b, 0x27, 0x0c, 0x00, 0x92, 0x05, 0x9a, 0x04, 0x9c, 0x9d, 0x7b, 0x68,
  ^~~~
```

**Comment by katelynsills** [comment]
Rerunning makes sense, but seems to make it harder to be able to easily evaluate and merge PRs. Do we need this test to pass before merging PRs?

**Comment by michaelfig** [comment]
I hate to take a step backwards, but that's my question too.  Apparent XS bugs are making me reluctant to merge PRs that are otherwise kosher.

**Comment by dckc** [comment]
To whom is the question directed? I don't have strong feelings, other than being strongly unhappy that gcc fails intermittently.

**Comment by michaelfig** [comment]
Directed to higher authority.  Ideally, we'd submit these reports to moddable or diagnose them ourselves, but it's a chore. :stuck_out_tongue_closed_eyes:

**Comment by dckc** [comment]
fixed in #2194 ; we don't have those great big arrays in our build process any longer.
